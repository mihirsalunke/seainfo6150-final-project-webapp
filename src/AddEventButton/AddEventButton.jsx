import React from "react";
import styles from "./AddEventButton.module.css";

const AddEventButton = () => {
    return (
        <div className={styles.addEventButton}>
            Add Event
        </div>
    );
};

export default AddEventButton;