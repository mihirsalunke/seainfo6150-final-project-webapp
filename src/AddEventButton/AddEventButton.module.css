@import url("https://fonts.googleapis.com/css2?family=Rowdies:wght@300&display=swap");

.addEventButton {
    background: orange;
    border-radius: 10px;
    color: #fff;
    padding: 10px;
    font-family: Rowdies, sans-serif;
    font-size: 1.5em;
    margin: 0 0 20px;
    width: 5em;
    text-align: center;
}

.addEventButton:hover {
    /* transform: translateY(2px); */
    background: green;
}

.addEventButton:active {
    /* transform: translateY(2px); */
    background: gray;
}