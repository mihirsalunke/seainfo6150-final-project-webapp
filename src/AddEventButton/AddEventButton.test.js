import React from "react";
import AddEventButton from "./AddEventButton.jsx";
import { render } from "@testing-library/react";

describe("AddEventButton tests", () => {
    it("renders correctly", () => {
        const { container } = render(<AddEventButton />);
        expect(container).toMatchSnapshot();
    });
});