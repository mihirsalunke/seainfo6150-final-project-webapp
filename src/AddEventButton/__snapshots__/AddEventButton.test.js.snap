// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddEventButton tests renders correctly 1`] = `
<div>
  <div
    class="addEventButton"
  >
    Add Event
  </div>
</div>
`;
